packages/dev/docs/src/addFocusRing.js

```diff
@@ -0,0 +1,37 @@
+import {FocusRing} from '@react-aria/focus';
+import {Link} from '@react-spectrum/link';
+import linkStyles from '@adobe/spectrum-css-temp/components/link/vars.css';
+import React from 'react';
+import ReactDOM from 'react-dom';
+import sideNavStyles from '@adobe/spectrum-css-temp/components/sidenav/vars.css';
+
+export function addFocusRing() {
+  let links = document.getElementsByClassName(sideNavStyles['spectrum-SideNav-itemLink']);
```

Comment:
for the sidenav links does it make sense to add this code here? https://github.com/adobe-private/react-spectrum-v3/blob/master/packages/dev/docs/src/Layout.js#L234-L240

or is this something that only should happen clientside?

Reply:
I think this has to render client side for the React-Spectrum Link to be available. @snowystinger?

Reply:
yeah, these things won't do anything if they are ssr'ed, no interactions, just dom. so to get the focus ring to work, it needs to have actual runnable code on the client, hence this

Reply:
got it. bummer.